### DISCUSSION TITLE
XPU in Solaris and exporting scene files?

### DISCUSSION THREAD

**Discussion by BrianHanke** [dreamworksanimation/openmoonray#82] [Q&A] [answered]
Howdy all, now that I've got MoonRay running in Houdini it's question time!

1. Is it possible to enable XPU mode in Solaris?
2. Is it possible to export a MoonRay scene file (rdla/b) from Solaris for command line rendering?

Thanks!

**Comment by scegielski** [comment, marked as answer]
1. There isn't currently an option to enable vector or XPU mode inside Solaris.
2. There is an "rdla output" field in the display options that will do what you want
